com.beatwaves.responsible/Samples~/ResponsibleGame/PlayModeTests/BddStyleTests.cs

```diff
@@ -1,36 +1,36 @@
-using System.Collections;
 using System.Linq;
 using NUnit.Framework;
 using Responsible;
-using UnityEngine.TestTools;
+using Responsible.Bdd;
 using static Responsible.Bdd.Keywords;
 using static Responsible.Responsibly;
 
 namespace ResponsibleGame.PlayModeTests
 {
 	// Example of BDD-style tests
+	[Feature("Restarting the game")]
 	public class BddStyleTests : SystemTest
 	{
-		[UnityTest]
-		public IEnumerator TriggerKey_Restarts_AfterFailing()
+		[Scenario("The game should be restartable easily after failure")]
+		public IBddStep[] Restart() => new[]
+		{
+			Given("the player has failed", this.FailTheGame()),
+			When("the user presses the trigger key", this.MockTriggerInput()),
+			Then("the game should be restarted", this.AssertTheGameHasBeenRestarted())
+		};
```

Comment:
Vocabulary feedback:

When looking at `this.MockTriggerInput()` (also in your opening post), it looks like you're mocking a trigger. But you aren't mocking anything, you're triggering something for real. Now perhaps the thing that's you're triggering is a mock.

I'd argue that in this example it could better be called `this.SimulateTriggerKeyInput`.

Also; not sure if intended but you're using both "the player" and "the user" for the same actor here, which might not be necessary depending on the language of your team.

Reply:
Thanks, that all makes sense!

I should probably review the sample tests as a whole from a consistency and vocabulary perspective.